### PR TITLE
Jack modi

### PR DESCRIPTION
**Pull request by KetsuRyo**

### CODE REVIEW

File: src/utils/validators.js [hunk 1]
```diff
@@ -51,13 +51,16 @@ class BaseValidator {
...
+  static maxBioLength = 255;
+  static allowedThemes = ["light", "dark"];
 
-  static validateSignupInfo = (_email, _username, _password) => {
+  static validateSignupInfo = (_email, _username, _password, _bio, _theme) => {
     const email = this.validateString(_email, "email");
     const username = this.validateString(_username, "username");
     const password = this.validateString(_password, "password");
+    const bio = this.validateString(_bio, "bio");
+    const theme = this.validateString(_theme, "theme");
```
Comment (ZhangTerrence) [inline src/utils/validators.js:63]: We don't have an a bio and theme section in our signup form so that was why I left it out. Unless we're planning on adding those sections to form, I don't think this is necessary.

Comment (KetsuRyo) [reply src/utils/validators.js:63]: So we just remove this code?

Comment (ZhangTerrence) [reply src/utils/validators.js:63]: Yeah unless you're planning on adding the inputs in the handlebars and making sure it works.

File: src/utils/validators.js [hunk 2]
```diff
@@ -74,79 +77,53 @@ export class UserValidator extends BaseValidator {
...
   };
-
-  static validateCreateFriendRequestInfo = (_username) => {
-    const username = this.validateString(_username, "username");
-
-    return {
-      username: xss(username)
-    };
-  };
-  s;
```
Comment (ZhangTerrence) [inline src/utils/validators.js:149]: Also why did you remove these functions? The program would crash when the user logs in or updates their profile since the controller which uses the validators can't find these functions.

**Comment by ZhangTerrence** [comment]
Also just make sure to merge the newest commits from main fixing @relaxwenj 's PRs.
